src/test/resources/json/original_path_with_quotes_response.json

```diff
@@ -0,0 +1,7 @@
+{
+  "data": {
+    "getClientFileMetadata": {
+      "originalPath": "a\"pathwith/quotes\"in"
```

Comment:
You could add a single quote to the folder name to check that at the same time?

Reply:
Yeah, that makes sense, I've added in two, one in the folder name and one in the filename.